pipeline/notifications/check_message.py

```diff
@@ -0,0 +1,25 @@
+from typing import List
+
+import prefect
+from prefect import task
+
+from pipeline.types import Notification, User
+
+
+class Notifier:
```

Comment:
If you are going to have a common class that specific implementations inherit from, then save the general name `Notifier` for the parent class, and have each implementation use a specific name, like `NewMessageNotifier`.

Reply:
I had a little trouble understanding what you meant by this comment, but now I understand. It be more or less similar to how it is in pipeline/matching/generators. And I agree that be for the best going forward for future implementations.